src/components/new-request/search.vue

```diff
@@ -102,9 +102,7 @@
       <!-- Change Name flow -->
       <template v-else-if="isChangeName">
         <BusinessLookupFetch />
-        {{ isSocietyDisabled }}{{ showSocietiesInfo }}
-        <!--  :show-modal="isSocietyDisabled && showSocietiesInfo"-->
-        <SocietiesInfo type='name' />
+        <SocietiesInfo v-if="isSocietyDisabled && showSocietiesInfo" :type="'request a name for'" :showDialog="showSocietiesInfo"/>
```

Comment:
Looks good!

Please put each attribute on a different line. I don't know if that lint rule is used in this repo but we try to do that in the other UIs.

Reply:
Fixed, thanks!